Orange/widgets/settings.py

```diff
@@ -200,10 +191,7 @@ def _var_exists(cls, setting, value, attributes, metas):
             return False
 
         attr_name, attr_type = value
-        if attr_type >= 100:
-            attr_type -= 100
-        else:
-            cls._warn_about_str_var_settings(setting)
+        attr_type %= 100
```

Comment:
While `attr_type` will now always be > 100, let's play it safe for any possible unmigrated settings.

Reply:
Good idea.
Maybe a comment in the code (not just in the PR) would help future generations decipher why attr_type is being percentaged :)

Reply:
You're right it'd be cryptic. I added a comment.

Reply:
> Ah, this is the compatibility trick that allows for newer settings to work with older versions. Could you please this comment to the code? There should probably also be a test that confirms old and new values working.

I added a comment, but not a test. New settings won't work with older versions (I guess), the trick might help in other direction, but I wouldn't guarantee it with a test. :)